tools/cli/internal/changelog/outputfilter/squash.go

```diff
@@ -0,0 +1,211 @@
+package outputfilter
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"regexp"
+)
+
+var (
+	// identifierRegex matches text enclosed in single quotes.
+	// Example: it'll match "value" from the string "added the new required request property 'value'".
+	identifierRegex = regexp.MustCompile(`'([^']+)'`)
+)
+
+type SquashHandler interface {
+	Squash(entriesGroupedByOperationID map[string][]*Entry) []*Entry
+}
+
+func newSquashHandlers() map[string]func(map[string][]*Entry) ([]*Entry, error) {
+	return map[string]func(map[string][]*Entry) ([]*Entry, error){
```

Comment:
[q] where are the enum handlers?

Reply:
here https://github.com/mongodb/openapi/pull/148